securedrop-export/debian/postinst

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+# postinst script for securedrop-export
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see https://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+
+case "$1" in
+    configure)
+    
+    update-desktop-database /usr/share/applications
```

Comment:
I believe the mime database also needs to be updated as well, see https://github.com/freedomofpress/securedrop-workstation/blob/503fd6f0930a24ddc92d9cfae4fa502275b3362f/dom0/sd-export-files.sls#L67